### PR TITLE
Create main application

### PR DESCRIPTION
**Pull request by iamquang95**
I just create main application and the base view of website (very simple)
It's ready to demo website (without database or other applications such as question, users...)

### CODE REVIEW

**Comment by namnd95** [comment]
I think we need to follow PEP8, 4 spaces per indentation level instead of tab character.

**Comment by iamquang95** [comment]
Yeah, you are right, i will fix it and re-commit.

File: main/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+    <title>
+      {% block page_title %}
+	    Trang chủ
```
Comment (namnd95) [inline main/templates/base.html]: You should replace the tab character on line 7.

**Comment by namnd95** [comment]
You should use pep8 package to check if the code is following pep8

**Pull request merged**